hedera-mirror-importer/src/main/resources/db/migration/v1/V1.65.0__record_file_hash_index.sql

```diff
@@ -0,0 +1,2 @@
+create index if not exists record_file__hash_prefix
```

Comment:
drop the existing hash, and do 
```suggestion
create unique index if not exists record_file__hash on record_file (hash collate "C");
```
for historical reason, record_file.hash is the 96-character hash string so there is no storage overhead when making it full text search capable using `collate "C"`

Reply:
Great thanks, I have changed the index.

Reply:
old index `idx_file_data_hash_unq` is not dropped

Reply:
I see, I have dropped it.